examples/showcase/anatomy.py

```diff
@@ -36,37 +42,38 @@ def minor_tick(x, pos):
 ax.set_xlim(0, 4)
 ax.set_ylim(0, 4)
 
-ax.tick_params(which='major', width=1.0)
-ax.tick_params(which='major', length=10)
+ax.tick_params(which='major', width=1.0, labelsize=14)
+ax.tick_params(which='major', length=10, labelsize=14)
 ax.tick_params(which='minor', width=1.0, labelsize=10)
 ax.tick_params(which='minor', length=5, labelsize=10, labelcolor='0.25')
 
 ax.grid(linestyle="--", linewidth=0.5, color='.25', zorder=-10)
 
-ax.plot(X, Y1, c=(0.25, 0.25, 1.00), lw=2, label="Blue signal", zorder=10)
-ax.plot(X, Y2, c=(1.00, 0.25, 0.25), lw=2, label="Red signal")
-ax.plot(X, Y3, linewidth=0,
-        marker='o', markerfacecolor='w', markeredgecolor='k')
+ax.plot(X, Y1, c='C0', lw=2, label="Blue signal", zorder=10)
+ax.plot(X, Y2, c='C1', lw=2, label="Orange signal")
+ax.plot(X[::3], Y3[::3], linewidth=0, markersize=12,
+        marker='o', markerfacecolor='none', markeredgecolor='C4', markeredgewidth=2)
 
 ax.set_title("Anatomy of a figure", fontsize=20, verticalalignment='bottom')
-ax.set_xlabel("X axis label")
-ax.set_ylabel("Y axis label")
+ax.set_xlabel("X axis label", fontsize=14)
+ax.set_ylabel("Y axis label", fontsize=14)
 
-ax.legend(loc="upper right")
+ax.legend(loc="upper right", fontsize=14)
 
 
 def circle(x, y, radius=0.15):
     from matplotlib.patches import Circle
     from matplotlib.patheffects import withStroke
-    circle = Circle((x, y), radius, clip_on=False, zorder=10, linewidth=1,
-                    edgecolor='black', facecolor=(0, 0, 0, .0125),
-                    path_effects=[withStroke(linewidth=5, foreground='w')])
+    circle = Circle((x, y), radius, clip_on=False, zorder=10, linewidth=2.5,
+                    edgecolor=royal_blue + [0.6], facecolor=(0, 0, 0, .0),
+                    path_effects=[withStroke(linewidth=7, foreground=(1, 1, 1, 1))])
     ax.add_artist(circle)
 
 
 def text(x, y, text):
-    ax.text(x, y, text, backgroundcolor="white",
-            ha='center', va='top', weight='bold', color='blue')
+    ax.text(x, y, text, backgroundcolor=(1, 1, 1, 1), zorder=100,
```

Comment:
perhaps replace backgroundcolor by something like `bbox={"fc": (1, 1, 1, .75)}` (or even `{'fc': 'w'}`); the borderless, implicit white rectangles around text look a bit weird otherwise.

Reply:
good call - the stroke effect around the lettering works really well.  See the updated plot above...

Reply:
lgtm, but I guess I'll let @rougier do the approval here.